charts/descheduler/templates/deployment.yaml

```diff
@@ -27,6 +27,10 @@ spec:
       priorityClassName: {{ .Values.priorityClassName }}
       {{- end }}
       serviceAccountName: {{ template "descheduler.serviceAccountName" . }}
+      {{- with .Values.imagePullSecrets }}
```

Comment:
I'm not an expert on the helm chart syntax, how does `with` handle empty values? Everywhere else we have been using `if`, but if the behavior is the same then it's probably fine